src/client/helpers/validarCorreo.js

```diff
@@ -0,0 +1,21 @@
+export function validarEmail(valor) {
```

Comment:
nitpick: prefer naming functions in english. code is more maintainable that way as english tends to be the universal language for developers.